src/views/menuconfig/Menuconfig.vue

```diff
@@ -66,7 +66,12 @@ export default class Menuconfig extends Vue {
 
   get items() {
     if (this.searchString !== "") {
-      return filterItems(this.storeItems, this.searchString.toLowerCase());
+      let searchStrMatch = /^(?:CONFIG_)?(.+)/.exec(this.searchString);
+      let searchMatch =
+        searchStrMatch && searchStrMatch.length > 1
+          ? searchStrMatch[1].toLowerCase()
+          : this.searchString.toLowerCase();
+      return filterItems(this.storeItems, searchMatch);
```

Comment:
What about some config which is like `Something_BT_BLE` or `WIRELESS_WIFI_SSID` where user might want to have fuzzy search for all of these not just `CONFIG_` one... 

Maybe we could use 
```js
this.storeItem.filter((item) => item.match(searchString.toLowerCase()))
```

Reply:
It should work since regex first group is optional. Is it not ?

Reply:
This is working fine, my bad!